Formula/lmod.rb

```diff
@@ -0,0 +1,64 @@
+class Lmod < Formula
+  desc "Lua-based environment modules system to modify PATH variable"
+  homepage "https://www.tacc.utexas.edu/research-development/tacc-projects/lmod"
+  url "https://github.com/TACC/Lmod/archive/7.7.16.tar.gz"
+  sha256 "cf14183f9954bf0d4c22db4b7db3a6459c8cb607b9d1ff25e18e0989b48bf58b"
+
+  depends_on "lua"
+
+  resource "luafilesystem" do
+    url "https://github.com/keplerproject/luafilesystem/archive/v1_7_0_2.tar.gz"
+    sha256 "23b4883aeb4fb90b2d0f338659f33a631f9df7a7e67c54115775a77d4ac3cc59"
+  end
+
+  resource "luaposix" do
+    # Don't use v34.0.2, it's broken (at least on macOS)
+    url "https://github.com/luaposix/luaposix/archive/v34.0.1.tar.gz"
```

Comment:
Latest version is 34.0.4, please update. If it does not work, then we need a link to the bug report.

Reply:
34.0.2 was the latest when I wrote this, I believe, and I think the fix was in master. I'll try 34.0.4.

Reply:
Yes, 34.0.4 works.